libs/core/src/lib/checkbox/checkbox/checkbox.component.ts

```diff
@@ -235,6 +282,11 @@ export class CheckboxComponent implements ControlValueAccessor {
         return event.keyCode === 32;
     }
 
+    /** @hidden Determines event source based on key code */
+    private _isEnterKeyEvent(event: KeyboardEvent): boolean {
+        return event.keyCode === 13;
```

Comment:
Hardcoding hinders flexibility and adaptability for future changes. We should try to avoid it as much. as possible.

Reply:
yes, can use CDK keycodes instead

Reply:
it is core/checkbox, if you see at line 282. they have used event.keycode, so i kept in same format. if core team suggest the same then i will change it.

Reply:
Done